src/client/app/component/CommentList/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './style.scss';
+import SingleComment from '../SingleComment';
+
+class CommentList extends React.Component {
+  render() {
+    let allComments = [];
+    const zero = 0;
+
+    if (this.props.comments && this.props.comments.length > zero) {
```

Comment:
and here just use `this.props.comments.length` since if its `0` it will evaluate to `false` otherwise it will be `true`